unobot.py

```diff
@@ -426,6 +426,8 @@ def _render_colored_cards(cards, theme=THEME_NONE):
         return bold + ''.join(ret) + CONTROL_NORMAL
 
     def card_playable(self, card):
+        if self.drawn and card != self.drawn:
```

Comment:
The problem with this logic is that it does not work for wilds (W/WD4). The internal representation of an unplayed wild does not have a color, but the internal representation of a played wild does. If the drawn card is a wild variant, no cards will be playable and the player will be forced to pass despite having a legal card to play.